Monika After Story/game/chess.rpy

```diff
@@ -1407,177 +616,420 @@ label demo_minigame_chess:
                 return
 
             # otherwise jump to a new game
-            jump mas_chess_new_game_start
+            jump .new_game_start
 
         # otherwise we are in good hands
         else:
-            # TODO test this
-
             $ loaded_game = quicksaved_game
 
-            if ur_nice_today:
+            if failed_to_load_save:
                 # we successfully loaded the unfinished game and player did not
                 # cheat
                 m 1eua "We still have an unfinished game in progress."
-            m "Get ready!"
 
-label mas_chess_game_load_check:
+            m 1efb "Get ready!"
 
     if loaded_game:
-        # now figure out the player color
-        if loaded_game.headers["White"] == mas_monika_twitter_handle:
-            $ player_color = ChessDisplayable.COLOR_BLACK
-        else:
-            $ player_color = ChessDisplayable.COLOR_WHITE
-        jump mas_chess_game_start
+        python:
+            is_player_white = mas_chess._get_player_color(loaded_game)
 
-label mas_chess_new_game_start:
-    # otherwise, new games only
-    if persistent._mas_chess_timed_disable is not None:
-        call mas_chess_dlg_chess_locked from _mas_chess_dclngs
-        return
+            practice_mode = loaded_game.headers.get("Practice", False)
+
+        jump .start_chess
+
+    label .new_game_start:
+        pass
+
+    #NOTE: ADD GAME MODES HERE
+    m 1eua "What game mode would you like to play?{nw}"
+    $ _history_list.pop()
+    menu:
+        m "What game mode would you like to play?{fast}"
+
+        "Normal chess.":
+            $ do_really_bad_chess = False
+
+        "Really bad chess.":
```

Comment:
`Really bad chess` is a trademarked name, so I think we should publicly/officially call this something else. Maybe `Random chess`?  Definitely not `Chaotic chess` since its still a chess game.

Regarding "really bad chess" in the code, we're probably safe if we leave it as is.